docs/articles/vs-test-adapter/AdapterV4-Release-Notes.md

```diff
@@ -1,5 +1,13 @@
 # Adapter V4 Release Notes
 
+## NUnit3 Test Adapter for Visual Studio - Version 4.2.1 - Jan 21, 2022
+
+This is a hotfix release for
+
+* [941](https://github.com/nunit/nunit3-vs-adapter/issues/941) When running `dotnet test` with filter, all tests are run and all console output is logged.
+
+This issue applies to test projects where all tests are excluded by a test item filter, then the opposite actually happens, and all these tests are being run. Note that this issue does not affect category filters.
```

Comment:
```suggestion
This fix applies to test projects where all tests are excluded by a test item filter. Previously, as of v4.2.0, all tests were unintentionally run. Note that this issue did not affect category filters.
```

Suggested language change, including which version the issue showed up in (I assume v4.2.0)